packages/styles/scss/utilities/_tooltip.scss

```
@@ -0,0 +1,372 @@
+//
+// Copyright IBM Corp. 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@use 'z-index';
+@use 'box-shadow';
+@use 'convert';
+
+// Does this need to be relative?
+@use 'type';
```

Yeah, I think it will in order to resolve the type entrypoint unfortunately.

```suggestion
@use '../../type';
```